samples/drivers/video/capture_to_lvgl/boards/stm32h7b3i_dk.overlay

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2024 Charles Dias <charlesdias.cd@outlook.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+/delete-node/ &sram1;
+/delete-node/ &sram2;
+
+&sram0 {
+	reg = <0x24000000 DT_SIZE_K(1024)>;
+};
```

Comment:
Please add comments to explain what this does and why it's needed as it's not necessarily obvious

Reply:
Done.